README.md

```diff
@@ -13,6 +13,10 @@ The frontend of this project is written in Angular and displays among other the
 
 To install nin simply run the command `npm install -g ninjadev-nin`.
 
+## Requirements
+
+This projects requires node version `7.1.0` or newer.
```

Comment:
Do we want to be this strict? I mean, it's fine by me, since it's already in homebrew, but it's only released 5 days ago so other platforms might not get it in packaging anytime soon.

Node 6.9, however is LTS and might be a better baseline. https://nodejs.org/en/blog/release/v6.9.0/